Subject: fix path for debian build
Description: Debian build is not done in configure directory,
 need to update way to get relative configure.orig script
Author: Olivier Sallou <osallou@debian.org>
Forwarded: no
Last-Updated: 2014-06-06
--- a/c++/configure.orig
+++ b/c++/configure.orig
@@ -1,3 +1,4 @@
 #!/bin/sh
-srcdir=`dirname $0`
+srcdir=$(readlink -f $0)
+srcdir=`dirname $srcdir`
 exec $srcdir/src/build-system/configure --srcdir=$srcdir ${1+"$@"}
